packages/website/docs/auxiliary/XYLabels.mdx

```diff
@@ -0,0 +1,562 @@
+```
+
+## Types
+
+### XYLabelPositioning Enum
+```ts
+enum XYLabelPositioning {
+  AbsolutePx = 'absolute_px',
+  AbsolutePercentage = 'absolute_percentage', 
+  DataSpace = 'data_space',
+}
+```
+
+### Configuration Interface
+```ts
+interface XYLabelsConfigInterface<Datum> extends XYComponentConfigInterface<Datum> {
+  // Required
+  y: NumericAccessor<Datum>;
+  
+  // Content
+  label?: StringAccessor<Datum>;
+  
+  // Positioning  
+  x?: NumericAccessor<Datum>; // Inherited from XYComponentConfigInterface
+  xPositioning?: GenericAccessor<XYLabelPositioning | string, Datum>;
+  yPositioning?: GenericAccessor<XYLabelPositioning | string, Datum>;
+  
+  // Styling
+  color?: ColorAccessor<Datum>;
+  backgroundColor?: ColorAccessor<Datum>;
+  labelFontSize?: NumericAccessor<Datum>;
+  labelTextBrightnessRatio?: number;
+  cursor?: StringAccessor<Datum>;
+  
+  // Clustering
+  clustering?: boolean;
+  clusterLabel?: StringAccessor<XYLabel<Datum>[]>;
+  clusterFontSize?: NumericAccessor<XYLabel<Datum>[]>;
+  clusterBackgroundColor?: ColorAccessor<XYLabel<Datum>[]>;
+  clusterCursor?: StringAccessor<XYLabel<Datum>[]>;
+  clusterLabelColor?: ColorAccessor<XYLabel<Datum>[]>;
+}
+```
+
+## Events
+The XYLabels component supports interactive events on both individual labels and clusters. Available selectors:
+
+- `XYLabels.selectors.label` - Events for individual labels
+- `XYLabels.selectors.cluster` - Events for label clusters (when clustering is enabled)
+
+```ts
+import { XYLabels } from '@unovis/ts'
+
+const events = {
+  [XYLabels.selectors.label]: {
+    click: (d: Datum) => alert(`Clicked: ${d.label}`),
+    mouseover: (d: Datum) => console.log('Label hovered:', d),
+    mouseleave: (d: Datum) => console.log('Label unhovered:', d)
+  },
+  [XYLabels.selectors.cluster]: {
+    click: (cluster: XYLabel<Datum>[]) => {
+      const labels = cluster.map(d => d.label).join(', ')
+      alert(`Cluster clicked! Contains: ${labels}`)
+    },
+    mouseover: (cluster: XYLabel<Datum>[]) => {
+      console.log('Cluster hovered:', cluster.length, 'labels')
+    }
+  }
+}
+```
+
+### Interactive Labels
+Click on any label below to see event handling in action:
+
+<XYWrapper
+  {...xyLabelsProps()}
+  containerProps={{
+    xDomain: [0, 9],
+    yDomain: [0, 12]
+  }}
+  components={[
+    { name: 'Axis', props: { type: 'x', gridLine: true }, key: 'xAxis' },
+    { name: 'Axis', props: { type: 'y', gridLine: true }, key: 'yAxis' },
+    { name: 'Area', props: { x: d => d.x, y: d => d.y, color: '#3B82F6', opacity: 0.2 }, key: 'area' },
+    { name: 'Line', props: { x: d => d.x, y: d => d.y, color: '#3B82F6', strokeWidth: 2 }, key: 'line' },
+    { name: 'XYLabels', props: { 
+      x: d => d.x, 
+      y: d => d.y + 0.5,
+      label: d => d.label,
+      color: '#1E40AF',
+      backgroundColor: 'rgba(255, 255, 255, 0.95)',
+      cursor: 'pointer',
+      events: {...events}
+    }, key: 'xyLabels' },
+  ]}
+  events={{}}
+/>
+
+### Cluster Events
```

Comment:
Do we need this? seems very similar to the interactive one. If you want to keep it, I'd somehow showcase, when you click on the cluster, it will return an array with all the information in the cluster.